src/components/ecs-service.ts

```diff
@@ -342,6 +356,27 @@ export class EcsService extends pulumi.ComponentResource {
       throw Error('Incorrect EcsService size argument');
     });
 
+    const fileSystemId = this.createPersistentStorage(argsWithDefaults).id;
+
+    const accessPoint = new aws.efs.AccessPoint(
+      `${this.name}-efs-ap`,
+      {
+        fileSystemId,
+        posixUser: {
+          uid: 1000,
+          gid: 1000,
+        },
+        rootDirectory: {
+          path: '/data',
+          creationInfo: {
+            ownerUid: 1000,
+            ownerGid: 1000,
+            permissions: '0755',
```

Comment:
Should we put this in a variable so it has a name? For example `READ_PERMISSIONS`.